src/Compilers/Lua/Portable/Syntax/SyntaxKind.cs

```diff
@@ -679,7 +694,28 @@ public enum SyntaxKind : ushort
         StatementList = 2072,
         EmptyStatement = 2073,
 
-        // Big gap 2086-3001 (insert new nodes here)
+        // Small gap 2074-2085
+
+        // Types
+        TypeBinding = 2086,
+        NullableType = 2087,
+        TableType = 2088,
+        TupleType = 2089,
+        VarargType = 2090,
+        TypeCast = 2091,
+        TypeUnion = 2092,
+        TypeIntersection = 2093,
+        Generic = 2094,
+        Type = 2095,
+        TableElementType = 2096,
+        TableTypeProperty = 2097,
+        TableTypeIndexer = 2098,
+        UnkeyedTableType = 2099,
+        TypeParameterListSyntax = 2100,
+        TypeArgumentListSyntax = 2101,
+
+        // Big gap 2105-3001 (insert new nodes here)
```

Comment:
The last new node seems to be `2101` but the gap says the next free one is `2105`?

Reply:
There will be more nodes so I didn't change it yet